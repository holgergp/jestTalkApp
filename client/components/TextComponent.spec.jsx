import React from "react";
import Text from "./TextComponent";
import renderer from "react-test-renderer";

describe("TextComponent", () => {
  it("renders", () => {
    const renderedApp = renderer.create(<Text value="test" />).toJSON();
    expect(renderedApp).toMatchSnapshot();
  });
});
