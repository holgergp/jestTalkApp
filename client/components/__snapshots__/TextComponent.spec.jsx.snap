// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextComponent renders 1`] = `
<div
  className="Text"
>
  test
</div>
`;
